documentation/docs/examples/cad-2doverlay.mdx

```diff
@@ -13,118 +13,152 @@ that follows the object it's attached to. Note that these overlays always will b
 they will not be hidden when the object is obscured by other objects.
 To achieve this it's necessary to use custom 3D objects.
 
+:::note
+There should be a limited number of HTML overlays as it can impact performance negatively if there's too many.
```

Comment:
It is not clear to me if this implies that one should limit the number of HTML elements added to a HTMLOverlayTool, or number of HTMLOverlayTool instances (or both).

Reply:
Clarified.